### PR TITLE
RE: Add new vulnerabilities to database

### PR DESCRIPTION
**Pull request by snoopysecurity**
Added 2 new vulnerabilities to database. Please review and change if needed. Thanks.

### CODE REVIEW

File: db/75-guessable-credentials.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "id": 75, 
+  "title": "Guessable credentials", 
+  "severity": "low", 
+  "description": [
+    "Web Applications often have predictable usernames and weak password policies set.", 
+    " This can easily allow an attacker to gain unauthorised access into the application",
```
Comment (andresriancho) [inline db/75-guessable-credentials.json]: unauthorized

File: db/76-x-content-type-options_header_missing.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "id": 76, 
+  "title": "X-Content-Type-Options header missing", 
+  "severity": "low", 
+  "description": [
+    "'X-Content-Type-Options' is a type of HTTP header that can be used to prevent MIME", 
+    "content-sniffing attacks in Internet Explorer and Google Chrome. MIME content-sniffing", 
+    "is a mechanism that allows browsers to inspect and dynamically guess the content",
+    "type and file type.\n\nMIME Sniffing checking algorithm has known problems which",
+    "might allow users to upload files that might contain malicous code.If an attacker"
```
Comment (andresriancho) [inline db/76-x-content-type-options_header_missing.json]: space after `.` in `code.If`

**Comment by andresriancho** [comment]
The CI system is telling me that your files are not valid JSON:

```
FAIL: test_all_files_JSON_content (tests.test_all_json.TestAllFilesAreJSON)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/data/tests/test_all_json.py", line 31, in test_all_files_JSON_content
    self.assertEqual(not_json, [])
AssertionError: Lists differ: ['75-guessable-credentials.jso... != []

First list contains 2 additional elements.
First extra element 0:
75-guessable-credentials.json

+ []
- ['75-guessable-credentials.json',
-  '76-x-content-type-options_header_missing.json']
```

https://circleci.com/gh/vulndb/data/51

**Comment by andresriancho** [comment]
Thanks for contributing! :+1: 

Please review the comments I've made and then we'll merge this into master.

**Comment by snoopysecurity** [comment]
Hey, any tips on how to rewrite as valid json? Is the syntax wrong? Apologies, I am new to Github.

**Comment by andresriancho** [comment]
To make changes to this pull request, just change your local files, commit and then push.

To identify where the json syntax is broken use any online tool that can analyze json; most likely it's a comma missing/extra somewhere

**Comment by snoopysecurity** [comment]
Managed to fix all errors. Thanks for the advice.

**Comment by andresriancho** [comment]
Thanks mate!

:+1: :dancer: :rocket:

**Pull request merged**